aries_vcx/src/common/proofs/prover/prover_internal.rs

```diff
@@ -114,6 +115,11 @@ pub fn credential_def_identifiers(credentials: &str, proof_req: &ProofRequestDat
 
                 let tails_file = value["tails_file"].as_str().map(|x| x.to_string());
 
+                let mut revealed = true;
```

Comment:
This block of added code can be done more safer, simpler:
```
let revealed = value["credential"]["cred_info"]["revealed"].as_bool().unwrap_or_default(true);
```
Although your code seems fine, this 1 liner is less error prone for 3 reasons:
- no `mut`
- no risk of mismatching strings in conditions and assignment, no duplication
- no panicking `unwrap()`